public/js/utils/source-map-worker.js

```diff
@@ -22,21 +33,21 @@ function enableSourceMaps() {
   sourceMapsEnabled = true;
 }
 
-function _resolveSourceMapURL(source) {
-  if (path.isURL(source.sourceMapURL) || !source.url) {
+function _resolveSourceMapURL(source: Source) {
+  const { url, sourceMapURL } = source;
```

Comment:
Thanks. I think the decision was to avoid destructuring, but still use defaults below.